fictadvisor-web/src/components/common/ui/form/dropdown/Dropdown.tsx

```diff
@@ -50,11 +52,15 @@ const Dropdown: FC<DropdownProps> = ({
   const handleChange = (_: SyntheticEvent, option: DropDownOption | null) => {
     onChange(option?.id || '');
   };
+  const handleInputChange = (text: string) => {
+    setIsEditing(true);
+    onInputChange(text);
+  };
 
-  const selectedValue: DropDownOption | null = useMemo(
-    () => options.find(({ id }) => id === value) || null,
-    [options, value],
-  );
+  const selectedValue: DropDownOption | null = useMemo(() => {
+    if (isEditing) return { label: value } as DropDownOption;
```

Comment:
use ternary operator instead of if else and remove return (use arrow function one line return)